sdk/keyvault/pom.xml

```diff
@@ -77,6 +77,7 @@
         <module>microsoft-azure-keyvault-cryptography</module>
         <module>microsoft-azure-keyvault-extensions</module>
         <module>microsoft-azure-keyvault-test</module>
+        <module>azure-security-test-keyvault-jca</module>
```

Comment:
Should we update file sdk/keyvault/tests.yml to add the tests module as well

Reply:
Ok, I will change it in the next commit.